script/Ciel/Day 7/07_01_CIEL07_2_1AFTER.txt

```diff
@@ -311,39 +329,44 @@ I'll let her believe that for now.
 [sha:ce9f9cf5902c65ef1b3dbceb9215d8ec41635ffa]{
 -- Page 260, Offset 15416.
 -- ……でも、本当にそうだったらいいとも思う。
-Still, it'd be nice if that was really the case.
+Still... It'd be nice if that really were the case.
 }
 [sha:680dbc05e92c570afa55683b8b4aadc96ced9690]{
 -- Page 260, Offset 15417.
 -- 受け取った時は随分な形見分けだと思ったけど、あれが何かしら特別なものだったのなら、あの父親とも何らかの繋がりがあったのだと思えるからだ。
-When I received it, I thought it was an awful keepsake to be granted,
- but if it really was something special,
- that would make it likely that it had some kind of special connection with my father.
+When I received this knife, I remember thinking that it was a paltry keepsake.
+ If it were actually something special,
+ I'd have been able to think of it as something that connected me with my father.
 }
 [sha:fb7c1bf13a6490bd0e6bfb0b6a468707d3e57398]{
 -- Page 261, Offset 15418.
 -- 「……コホン。ちょっと話がズレましたね。
-"%{i}Ahem.%{/i} We've gotten a little off-topic.
+"%{i}Ahem%{/i}. We've gotten a little off topic.
 }
 [sha:df19a9589b61837478c6e7612095cc8adb2a829d]{
 -- Page 261, Offset 15419.
 -- わたしの説明は以上です。遠野くんの方はどうですか？　あれから体に異常はありませんか？」
-That's all for my explanations. What about you? Nothing strange going on in your body?"
+That's all for my explanations.
+ What about you?
+ How has your body been holding up?"
```

Comment:
Ciel is tentatively trying to probe him about Roa, I feel like the sentence should be a bit more odd, since she asks 体に異常はありませんか instead of the more common 体の調子はどう？